packages/ogm/src/types.ts

```diff
@@ -1 +1,6 @@
 export { GraphQLOptionsArg, GraphQLWhereArg, DeleteInfo, GraphQLSortArg } from "@neo4j/graphql";
+
+export type DriverConfig = {
```

Comment:
Why does this need to exist? Is it not already available to users because OGMConstructor === Neo4jGraphQLConstructor, and that contains this type definition?

Reply:
IDK it's gone now!